text/message_verification.go

```diff
@@ -48,6 +47,15 @@ func NewVerificationEmailSent() *Message {
 	}
 }
 
+func NewVerificationConfirmation() *Message {
```

Comment:
Please move this to `message_login` - this file should stay unmodified. `message_verification` refers to the self-service verification (e.g. email verification flow). Thank you :)

Reply:
@aeneasr Thanks for the feedback. Changes has been moved to `message_login`. Hope I did it right this time :)